buildSrc/src/main/java/org/elasticsearch/gradle/test/rest/YamlRestCompatTestPlugin.java

```diff
@@ -20,6 +20,7 @@
 package org.elasticsearch.gradle.test.rest;
```

Comment:
I missed a spot. I think this should be `internal` as well.

Reply:
done on https://github.com/elastic/elasticsearch/pull/67394/commits/7c5be906d0306af47e0646c49a835215cb342d14 , had to add getters/setters since alot of access was via package private directly.